cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -817,15 +817,11 @@
     /** Twill Runnable configuration **/
     public static final String CONTAINER_VIRTUAL_CORES = "explore.executor.container.num.cores";
     public static final String CONTAINER_MEMORY_MB = "explore.executor.container.memory.mb";
-    public static final String CONTAINER_INSTANCES = "explore.executor.container.instances";
```

Comment:
@awholegunch that file doesn't really matter, it's for development purposes only